src/charts/utilization-bar/utilization-bar-chart.component.js

```diff
@@ -49,6 +51,11 @@
        <label class="label-title">Default Layout, no Thresholds</label>
        <pf-utilization-bar-chart chart-data=data1 chart-title=title1 units=units1></pf-utilization-bar-chart>
        <br>
+       <label class="label-title">layout='inline', tooltip-function='customTooltip'</label>
```

Comment:
Could you just title this "Inline layout, custom tooltips"

Reply:
@jeff-phillips-18 want me to change the other one as well?

Reply:
Please.